<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelListaRozmowcow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;
background-color:white;
border-top-left-radius: 5px;
color: black;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Lista rozmówców&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUsun">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
    </property>
    <property name="text">
     <string>Usuń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZablokuj">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
    </property>
    <property name="text">
     <string>Zablokuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOdblokuj">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
    </property>
    <property name="text">
     <string>Odblokuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartSerwera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
    </property>
    <property name="text">
     <string>START SERWERA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopSerwera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#82868c;
border: 2px solid black;
color: black;
</string>
    </property>
    <property name="text">
     <string>STOP SERWERA</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetUzytkownicy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-left-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerwerInfo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 56px;
border-radius: 20px;
background-color: red;
border: 1px solid black;
color: black;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Wyłaczony&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelListaRozmowcow_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;
background-color:white;
border-top-right-radius: 5px;
color: black;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Lista zablokowanych&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetZablokowani">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>256</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-right-radius: 5px;
background-color:#d3dae5;
color: black;
border: 1px solid black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
